### PR TITLE
added wget line for downloading install script and formatting updates

### PR DESCRIPTION
**Pull request by dsarahstamps**

### CODE REVIEW

File: doc/install/stampede.tex [hunk 1]
```diff
@@ -13,32 +13,30 @@
 
 
 
-Setting up ASPECT on TACC Stampede is much easier than on previous systems since
+\noindent Setting up ASPECT on TACC Stampede is much easier than on previous systems since
 most of the necessary libraries are already correctly installed and configured. Following all
 the instructions in this document should allow you to start running ASPECT within 30
-minutes.
+minutes. For further information on Stampede you can also refer to the user guide at \\\url{http://www.tacc.utexas.edu/user-services/user-guides/stampede-user-guide} \\
```
Comment (bangerth) [inline doc/install/stampede.tex]: How come you need the 3 backslashes?

File: doc/install/stampede.tex [hunk 2]
```diff
@@ -13,32 +13,30 @@
 
 
 
-Setting up ASPECT on TACC Stampede is much easier than on previous systems since
+\noindent Setting up ASPECT on TACC Stampede is much easier than on previous systems since
```
Comment (bangerth) [inline doc/install/stampede.tex]: I would suggest you simply follow the formatting in the rest of the document, including paragraph indents. Or do you use `\noindent` for a purpose?

File: doc/install/stampede.tex (hunk 1 above)
Comment (dsarahstamps) [inline doc/install/stampede.tex]: // is to make sure the url goes to the next line. Perhaps putting a space between "//" and /url is easier to understand.

File: doc/install/stampede.tex (hunk 2 above)
Comment (dsarahstamps) [inline doc/install/stampede.tex]: Justified indentation is a personal preference. No problem to change use the defaults.

File: doc/install/stampede.tex (hunk 1 above)
Comment (bangerth) [inline doc/install/stampede.tex]: Ah, yes. Space sounds good.

File: doc/install/stampede.tex (hunk 2 above)
Comment (bangerth) [inline doc/install/stampede.tex]: I usually think that just using the default style is the way to go. If you think about having a paragraph break but not use an indent, maybe no paragraph break is necessary to begin with. But then I'm not feeling strongly enough about it to put on a fight :-)

**Comment by dsarahstamps** [comment]
We've found that the wget command doesn't work. Is there a place that the script can be downloaded with wget?

**Comment by bangerth** [comment]
I think the correct address is https://raw.githubusercontent.com/geodynamics/aspect/master/doc/install/aspect-setup_stampede.sh

**Pull request merged**